src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string
+};
+
+export class App extends React.Component< {}, State > {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.key);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.key);
+  }
+
+  key = (event: KeyboardEvent) => (
+    this.setState({ pressedKey: event.key })
```

Comment:
let's check if the key is `space` (with `event.code`) and write `[space]` if so
<img width="187" alt="image" src="https://user-images.githubusercontent.com/55802630/183034705-7927493b-8d7f-4ded-beed-72a1309c93bd.png">